apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -355,6 +359,15 @@ where
                                 );
                             }
                         }
+                        // should we update the bridge pool nonce?
+                        if !increment_bp_nonce
```

Comment:
This isn't quite right, it will increase the nonce too frequently. We should increase the nonce every time a `TransferToEthereum` event gets enough votes to be confirmed.